woocommerce/woocommerce-android
# Hackweek. Improved autofill for the login

### Description

The goal of the PR is to make it less frictionless to use password managers for entering wp.com credentials 

### Testing instructions

* Do normal wp.com login flow with your password manager of choice (I used 1passwpord)
* Notice that on both login/password screen now you have to do fewer clicks in order to enter your credentials

### Images/gif

### BEFORE|AFTER



https://user-images.githubusercontent.com/4923871/192969359-c09daad8-e573-4083-b7d8-b794638f4570.mp4


https://user-images.githubusercontent.com/4923871/192968909-4877c2a5-416e-4143-988e-38eaa77ef71d.mp4
